lib/macho/headers.rb

```diff
@@ -50,6 +56,8 @@ module MachO
     CPU_TYPE_ANY => "CPU_TYPE_ANY",
     CPU_TYPE_I386 => "CPU_TYPE_I386",
     CPU_TYPE_X86_64 => "CPU_TYPE_X86_64",
+    CPU_TYPE_ARM => "CPU_TYPE_ARM",
+    CPU_TYPE_ARM64 => "CPU_TYPE_ARM64",
```

Comment:
My main motivation for leaving it like that was matching `libmacho`'s constant names, since those are used throughout `otool`.

I agree that they're very ugly and unwieldy, and I think that replacing them with shorthands is a good idea. Backwards compatibility shouldn't be a *huge* problem, as the library is still fairly immature and had broken compatibility in some aspect with virtually every release.

I'll get to work on this :+1: